scrypto/src/resource/resource_builder.rs

```diff
@@ -128,20 +119,54 @@ impl FungibleResourceBuilder {
         self.build(None).0
     }
 
+    pub fn initial_supply_with_owner<T: Into<Decimal>>(&self, amount: T) -> (Bucket, Bucket) {
```

Comment:
I'd suggest we revert this user-facing API to `initial_supply` before we finalise on a decision to support two variants.